Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY : all clean
```

Comment:
I added a Makefile for libpfc, which is totally parallel to the meson/ninja stuff. My project is small and make-only, and it seems to be a pain to ask uses to install *two* other build tools just to build the libpfc part, so I added this to simplify the process for them. It pretty much does the same thing as ninja/meson, but outputs to a different location. If you want to keep this out of your branch that's fine, I'll try to cherry-pick the other commits in that case.